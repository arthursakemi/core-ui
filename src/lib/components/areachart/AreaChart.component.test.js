import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/areachart/__snapshots__"
});
