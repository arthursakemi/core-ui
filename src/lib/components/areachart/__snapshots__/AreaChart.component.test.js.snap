// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots AreaChart Default 1`] = `
<div
  className="sc-bwzfXH hilSGa"
>
  <h3
    className="sc-htpNat loHzkE"
  >
    Vege-Lite area chart
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-vegachart"
    id="vis_area_chart"
  />
</div>
`;
